src/components/Profile/VotedProposalsBox.tsx

```diff
@@ -25,8 +25,8 @@ function VotedProposalsBox({ address }: Props) {
       {isLoading && <SkeletonBars amount={votes.length || 3} height={83.5} />}
       {!isLoading &&
         (votes.length > 0 ? (
-          votes.map((vote) => {
-            return <ProfileProposalItem key={vote.id} votedProposal={vote} />
+          votes.map((vote, idx) => {
+            return <ProfileProposalItem key={vote.id + idx} votedProposal={vote} />
```

Comment:
Why vote.id isn't enough?

Reply:
I thought the same, but saw an error of elements with repeated id

<img width="1496" alt="image" src="https://github.com/decentraland/governance/assets/2858950/b60eedff-dd33-46af-bb42-10454362190f">